.travis.yml

```diff
@@ -3,13 +3,25 @@ cache: cargo
 sudo: required
 rust:
   - 1.33.0
+  - 1.36.0
```

Comment:
Is there any reason for testing only for 1.36 version? I think we should test the latest stable version overall. `stable` will check against the latest rust stable version.

Reply:
Cc @brainstorm

Reply:
No particular reason, feel free to change it ;)